src/core/MOM_barotropic.F90

```diff
@@ -1557,7 +1557,6 @@ subroutine btstep(U_in, V_in, eta_in, dt, bc_accel_u, bc_accel_v, forces, pbce,
       ubt_st(I,j) = ubt(I,j) 
     enddo ; enddo
   endif
-
   if (CS%id_vbtdt > 0) then
```

Comment:
This change removed a blank line, but the fix that actually was needed was to remove the spaces at the end of the two lines a few lines above.  Completely blank lines (without any space or tabs) are fine, but it is a MOM6 convention that there are no spaces after the last non-space character on any line.  To find trailing white space, try the command `grep -n ' $' src/*/*.F90 src/parameterizations/*/*.F90 config_src/*/*.F90` when sitting in your MOM6 directory.